IntGeVaP/IntGeVaP-C.R

```diff
@@ -0,0 +1,420 @@
+################################################################################
+# 01-annotate_variants.R
```

Comment:
I think it is fine to retain the original titles of the scripts, specifying CAVATICA, user-input. I don't really like the name of the tool being the name of the R script. Something like:
```
01-annotate_variants_CAVATICA_input.R
01-annotate_variants_custom_input.R
```

Is this accurate to say?